Octokit/Models/Response/Page.cs

```diff
@@ -9,22 +10,32 @@ public enum PagesBuildStatus
         /// <summary>
         ///  The site has yet to be built
         /// </summary>
+        [Parameter(Value = "null")]
         Null,
 
         /// <summary>
+        /// The build has been requested but not yet begun
+        /// </summary>
+        [Parameter(Value = "queued")]
+        Queued,
```

Comment:
I found this in the documentation, but not here, so just added it while I was there 😄